src/Browser.php

```diff
@@ -725,6 +728,74 @@ public function withResponseBuffer($maximumSize)
         ));
     }
 
+    /**
+     * The `withHeader(string $header, string $value): Browser` method can be
+     * used to add a request header for all following requests.
```

Comment:
I think something got mixed up when squashing the commits, in an older version we broke this down to:
```suggestion
     * Add a request header for all following requests.
```